target/scripts/startup/setup-stack.sh

```diff
@@ -95,9 +95,11 @@ function _setup_apply_fixes_after_configuration() {
   _log 'debug' 'Removing files and directories from older versions'
   rm -rf /var/mail-state/spool-postfix/{dev,etc,lib,pid,usr,private/auth}
 
-  # /tmp/docker-mailserver/rspamd/dkim
-  _log 'debug' "Ensuring ${RSPAMD_DMS_DKIM_D} is owned by '_rspamd:_rspamd'"
-  chown -R _rspamd:_rspamd "${RSPAMD_DMS_DKIM_D}"
+  if [[ -n ${RSPAMD_DMS_DKIM_D} ]] ; then
+    # /tmp/docker-mailserver/rspamd/dkim
+    _log 'debug' "Ensuring ${RSPAMD_DMS_DKIM_D} is owned by '_rspamd:_rspamd'"
+    chown -R _rspamd:_rspamd "${RSPAMD_DMS_DKIM_D}"
+  fi
```

Comment:
```suggestion
  _rspamd_get_envs
  # /tmp/docker-mailserver/rspamd/dkim
  _log 'debug' "Ensuring '${RSPAMD_DMS_DKIM_D}' is owned by '_rspamd:_rspamd'"
  chown -R _rspamd:_rspamd "${RSPAMD_DMS_DKIM_D}"
```

Reply:
Question, with your readonly usage, does it cause an issue if the function is called again later in setup? (which it presumably is by the rspamd scripts)

Reply:
And here I am thinking I'd know my own code... you're absolutely right. I will provide a fix for that, then we can merge this PR.

Reply:
When #3837 is merged, we can apply this suggestion and merge this PR.